test/specs/FindElementSpec.ts

```diff
@@ -3,15 +3,17 @@ import { BrowserUtils } from '../../src/commons/BrowserUtils';
 import { SelectorType } from '../../src/enums/SelectorType';
 import { describeCommon } from '../TestHelper';
 
+const IMAGE_XPATH: string = "//*[@id='content_images']/div/p/img";
 /**
  * wdio-allure-ts FindElement & FindElements tests
  */
 describeCommon('findElement of BrowserUtils Tests', () => {
```

Comment:
missing negative test -not existing element 
and test with multiple elements with same ID

Reply:
done